libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -3189,6 +3189,27 @@ bool AP_AHRS::get_velocity_NED(Vector3f &vec) const
     return state.velocity_NED_ok;
 }
 
+bool AP_AHRS::location_for_origin_offset(Location &loc, const Vector3F &offset_ned) const
```

Comment:
could we copy the comments from the .h to .cpp.  I know Jonathan never liked the duplicate comments but I find it convenient to have them in both places.

Reply:
Done